packages/lesmis/src/components/DashboardExportModal/utils/useSubDashboards.js

```diff
@@ -0,0 +1,42 @@
+import { useMemo } from 'react';
+import { useDashboardDropdownOptions } from '../../../utils/useDashboardDropdownOptions';
+import { useUrlParams } from '../../../utils';
+import { useDashboardData } from '../../../api';
+
+export const useSubDashboards = () => {
```

Comment:
I think there could be a better name than subDashboards because they are not part of dashboards. It could be exportableDashboards or something. Also see my main comment about moving this into the file with the dashboardModal. I think if it's in the same file then it will be more easy to understand what it is for.